docs/en/sql-reference/statements/kill.md

```diff
@@ -44,6 +73,11 @@ A test query (`TEST`) only checks the user’s rights and displays a list of que
 
 ## KILL MUTATION
 
+The presence of long-running or incomplete mutations often indicates that a ClickHouse service is running poorly. The asynchronous nature of mutations can cause them to consume all available resources on a system. You may need to either: 
+
+- Pause all new mutations, `INSERT`s , and `SELECT`s and allow the queue of mutations to complete.
+- Or manually kill some of these mutations by sending a `KILLSIG` command.
```

Comment:
What's KILLSIG?

Reply:
I'm assuming this was a typo, and @thomoco meant to write `SIGKILL`, correct Thom?

```suggestion
- Or manually kill some of these mutations by sending a `SIGKILL` command.
```

Reply:
No, it is not possible. This part was probably hallucinated.